components/utilities/data/KusamaAuctions.json

```diff
@@ -1174,27 +1174,45 @@
     "startBlock": 16128903,
     "startHash": "0xac9f5323ff9fcede5cf43a061f744de0f424962ccc8cb4e6889699a90416f7e9",
     "endPeriodBlock": 16155903,
-    "endPeriodHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "endPeriodHash": "0xc544915dff81662b91a147f2e2ee9bd2b252862aafc14fd947a6a7b2e89f14e1",
     "biddingEndsBlock": 16227903,
     "biddingEndsHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
     "onboardStartBlock": 15724800,
     "onboardStartHash": "0xfad9be21093dacbe4af3e56a3c854d77bf82ba498e5fbc786eb7970f92fd99c5",
     "onboardEndBlock": 20563200,
     "onboardEndHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
     "startDate": 1673350170000,
-    "endPeriodDate": null,
+    "endPeriodDate": 1673512758001,
     "biddingEndsDate": null,
     "onboardStartDate": 1670898876012,
     "onboardEndDate": null
   },
   {
     "index": 66,
     "startBlock": 16165188,
-    "startHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "startHash": "0x55a5db40171b479d0717082d39303d1c0c02908c46c9747812e38c5700777e52",
     "endPeriodBlock": 16192188,
     "endPeriodHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
     "biddingEndsBlock": 16264188,
     "biddingEndsHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "onboardStartBlock": 16329600,
+    "onboardStartHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "onboardEndBlock": 21168000,
+    "onboardEndHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "startDate": 1673568684001,
+    "endPeriodDate": null,
+    "biddingEndsDate": null,
+    "onboardStartDate": null,
+    "onboardEndDate": null
+  },
+  {
+    "index": 67,
+    "startBlock": 16265988,
```

Comment:
@DrW3RK first example of a newly scheduled auction automatically being added to the cache!